### PR TITLE
Added BYTES_TO_XX.fct files to utilis-1.0.0

### PR DESCRIPTION
**Pull request by MoritzO02**
Added BYTES_TO_DWORD and BYTES_TO_WORD under the new assemblies directorie in utils-typelib
for issue #221

### CODE REVIEW

File: data/typelibrary/utils-1.0.0/typelib/assembling/BYTES_TO_WORD.fct [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	<FunctionBody>
+		<ST><![CDATA[FUNCTION BYTES_TO_WORD : WORD
+
+VAR_INPUT
+	IN_00 : BYTE;    // BYTE 00
+	IN_01 : BYTE;    // BYTE 01
+END_VAR
+
+(* this Function combines the 2 BYTES to a word *)
+BYTES_TO_WORD := WORD_BYTE_ZZ;
```
Comment (MartinMelikMerkumians) [inline data/typelibrary/utils-1.0.0/typelib/assembling/BYTES_TO_WORD.fct]: see above comment

File: data/typelibrary/utils-1.0.0/typelib/assembling/BYTES_TO_DWORD.fct [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+VAR_INPUT
+	IN_00 : BYTE;    // BYTE 00
+	IN_01 : BYTE;    // BYTE 01
+	IN_02 : BYTE;    // BYTE 02
+	IN_03 : BYTE;    // BYTE 03
+END_VAR
+
+(* this Function combines the 2 BYTES to a word *)
+BYTES_TO_DWORD := DWORD_BYTE_ZZ;
+BYTES_TO_DWORD := BYTES_TO_DWORD OR SHL(BYTE_TO_DWORD(IN_03), SHFT_BYTE_03);
```
Comment (MartinMelikMerkumians) [inline data/typelibrary/utils-1.0.0/typelib/assembling/BYTES_TO_DWORD.fct]: The implementation can be simplified by using the bytewise access operator
e.g. `BYTES_TO_DWORD.%Bx` (or if thats not working on return values, a temporary variable)

**Review by MartinMelikMerkumians — CHANGES_REQUESTED** [review]
As a general remark, TO* named functions are reserved for official cast functions.
The functions introduced here are not casts and introduce a big potential for mistyping names BYTE_TO vs BYTES_TO

Please propose other names, in order to make those functions clearly distinguishable from standardized cast functions

File: data/typelibrary/utils-1.0.0/typelib/assembling/ASSEMBLE_DWORD_FROM_BYTES.fct [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+		<ST><![CDATA[FUNCTION ASSEMBLE_DWORD_FROM_BYTES : DWORD
+
+VAR_INPUT
+	IN_00 : BYTE;    // BYTE 00
+	IN_01 : BYTE;    // BYTE 01
+	IN_02 : BYTE;    // BYTE 02
+	IN_03 : BYTE;    // BYTE 03
+END_VAR
+
+(* this Function combines the 2 BYTES to a word *)
```
Comment (diplfranzhoepfinger) [inline data/typelibrary/utils-1.0.0/typelib/assembling/ASSEMBLE_DWORD_FROM_BYTES.fct]: comment to the top please

File: data/typelibrary/utils-1.0.0/typelib/assembling/ASSEMBLE_WORD_FROM_BYTES.fct [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	</InterfaceList>
+	<FunctionBody>
+		<ST><![CDATA[FUNCTION ASSEMBLE_WORD_FROM_BYTES : WORD
+
+VAR_INPUT
+	IN_00 : BYTE;    // BYTE 00
+	IN_01 : BYTE;    // BYTE 01
+END_VAR
+
+(* this Function combines the 2 BYTES to a word *)
```
Comment (diplfranzhoepfinger) [inline data/typelibrary/utils-1.0.0/typelib/assembling/ASSEMBLE_WORD_FROM_BYTES.fct]: comment to the top please.

**Review by diplfranzhoepfinger — APPROVED** [review]
looks good to me, Approved

**Review by franz-hoepfinger-4diac — APPROVED** [review]
Looks good to me, Approved.

**Review by franz-hoepfinger-4diac — APPROVED** [review]

**Review by azoitl — APPROVED** [review]

**Pull request merged**